src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java

```diff
@@ -216,6 +219,20 @@ private Job createJobExecutionUpdate(String jobId, JobExecution jobExecution) {
       }
     }
 
+    if (jobExecution.getJobInstance().getJobName().contains(AUTH_HEADINGS_UPDATES.getValue())) {
+      String fileName = ExecutionContextUtils.getFromJobExecutionContext(jobExecution, AUTHORITY_CONTROL_FILE_NAME);
+      if (StringUtils.isNotBlank(fileName)) {
+        result.setFileNames(singletonList(fileName));
+      }
+    }
+
+    if (jobExecution.getJobInstance().getJobName().contains(FAILED_LINKED_BIB_UPDATES.getValue())) {
+      String fileName = ExecutionContextUtils.getFromJobExecutionContext(jobExecution, AUTHORITY_CONTROL_FILE_NAME);
+      if (StringUtils.isNotBlank(fileName)) {
+        result.setFileNames(singletonList(fileName));
+      }
+    }
```

Comment:
Could be simplified to one `if` statement, also `jobExecution.getJobInstance().getJobName()` could be moved to a variable.

Reply:
Thanks, done